data/org.freedesktop.UDisks2.xml

```diff
@@ -1406,7 +1406,7 @@
         @size: The desired size of the partition, in bytes.
         @type: The type of partition to create (cf. the #org.freedesktop.UDisks2.Partition:Type property) or blank to use the default for the partition table type and OS.
         @name: The name for the new partition or blank if the partition table do not support names.
-        @options: Options (currently unused except for <link linkend="udisks-std-options">standard options</link>).
+        @options: Options - known options (in addition to <link linkend="udisks-std-options">standard options</link>) includes <parameter>partition-type</parameter> (of type 's').
```

Comment:
*known options...include...* makes sense to me. Don't really care about consistency here. :)